app/ocr.py

```diff
@@ -17,6 +17,8 @@
 from spacy.tokens.span import Span
 from spacy.tokens.token import Token
 from spacy.matcher import Matcher
+from spacy.matcher import PhraseMatcher

```

Comment:
Bundle `Matcher` and `PhraseMatcher` in one import line as they both originate from `spacy.matcher`.

```suggestion
from spacy.matcher import Matcher, PhraseMatcher
```